client/state/data-layer/wpcom/read/teams/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Internal dependencies
+ */
+import { READER_TEAMS_REQUEST, READER_TEAMS_RECEIVE } from 'state/action-types';
+import wpcom from 'lib/wp';
+
+export function handleTeamsRequest( store, action, next ) {
+	wpcom.req.get( '/read/teams', { apiVersion: '1.2' } )
+		.then(
+			payload => {
```

Comment:
well, payload conforms with FSA, so, yeah it's fine. I agree with Dennis that we may want a mapper that turns an API response into something calypso likes, but in this case, since it's already that, I think a comment would be fine.